services/openshift/scripts/openshift_provision

```diff
@@ -67,6 +67,51 @@ if [ ! -f ${ORIGIN_DIR}/configured.router ]; then
   oc expose service docker-registry --hostname ${registry_name}
 fi
 
+
+# Securing the Registry
+# https://docs.openshift.org/latest/install_config/install/docker_registry.html#securing-the-registry
+if [ ! -f ${ORIGIN_DIR}/secured.registry ]; then
+  # Public name for the registry
+  REGISTRY_ROUTE=$(oc get route docker-registry -o template --template='{{ .spec.host }}')
+
+  # Get IP and Port of the registry service
+  REGISTRY_SERVICE_IP=$(oc get svc/docker-registry -o template --template='{{ .spec.clusterIP }}')
+  REGISTRY_SERVICE_PORT=$(oc get svc/docker-registry -o template --template='{{ (index .spec.ports 0).port }}')
+
+  # Create certificates for registry
+  oadm ca create-server-cert --signer-cert=$OPENSHIFT_DIR/ca.crt \
+      --signer-key=$OPENSHIFT_DIR/ca.key --signer-serial=$OPENSHIFT_DIR/ca.serial.txt \
+      --hostnames="$REGISTRY_ROUTE,$REGISTRY_SERVICE_IP" \
```

Comment:
We should probably also add `docker-registry.default.svc.cluster.local` to hostnames.
Registry IP address is used internally by openshift, but this may change in future.